install-linux.sh

```diff
@@ -116,9 +118,9 @@ do_rpm_install(){
 do_dpkg_install(){
   get_package "deb"
   sleep 2
-  if dpkg -s cyral-sidecar > /dev/null 2>&1; then
+  if dpkg -s cyral-sidecar > /dev/null 2>&1 || dpkg -s sidecar > /dev/null 2>&1; then
     echo "Removing existing installation..."
-    dpkg -r cyral-sidecar > /dev/null 2>&1
+    dpkg -r cyral-sidecar sidecar > /dev/null 2>&1
```

Comment:
if one of these packages are not present in the system, wont this command fail?

Reply:
with the -r option it continues:
```
sudo dpkg -r cyral-sidecar sidecar
dpkg: warning: ignoring request to remove cyral-sidecar which isn't installed
(Reading database ... 67831 files and directories currently installed.)
Removing sidecar (4.14.6-alpha) ...
Removed "/etc/systemd/system/multi-user.target.wants/cyral-alerter.service".
Removed "/etc/systemd/system/multi-user.target.wants/cyral-authenticator.service".
Removed "/etc/systemd/system/multi-user.target.wants/cyral-certificate-manager.service".
Removed "/etc/systemd/system/multi-user.target.wants/cyral-dispatcher.service".
Removed "/etc/systemd/system/multi-user.target.wants/cyral-forward-proxy.service".
Removed "/etc/systemd/system/multi-user.target.wants/cyral-oracle-wire.service".
Removed "/etc/systemd/system/multi-user.target.wants/cyral-pg-wire.service".
Removed "/etc/systemd/system/multi-user.target.wants/cyral-push-client.service".
Removed "/etc/systemd/system/multi-user.target.wants/cyral-service-monitor.service".
Removed "/etc/systemd/system/multi-user.target.wants/cyral-sqlserver-wire.service".
```